src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -162,8 +161,6 @@ public static void Run(ListSourceArgs args, Func<ILogger> getLogger)
                                 source.IsEnabled ? string.Format(CultureInfo.CurrentCulture, Strings.SourcesCommandEnabled) : string.Format(CultureInfo.CurrentCulture, Strings.SourcesCommandDisabled)));
                             getLogger().LogMinimal(string.Format(CultureInfo.CurrentCulture, "{0}{1}", sourcePadding, source.Source));
                         }
-
-                        WarnForHttpSources(sourcesList, getLogger);
```

Comment:
I am actually thinking if it makes sense for us to still display warnings for HTTP sources in `list source` command.

Reply:
I think that's a good idea. I have added a new warning